ErsatzTV/Controllers/ArtworkController.cs

```diff
@@ -253,4 +255,70 @@ private async Task<IActionResult> GetEmbyArtwork(string path, CancellationToken
             });
 #endif
     }
+
+    [HttpGet("/iptv/logos/gen")]
+    public IActionResult Get(string text)
+    {
+        int logoWidth = 200;
```

Comment:
I would move this logic behind an interface (maybe similar to `ISongVideoGenerator`). You should be able to inject the interface in the controller constructor.